scripts/zones/Ifrits_Cauldron/mobs/Goblin_Alchemist.lua

```diff
@@ -2,12 +2,18 @@
 -- Area: Ifrit's Cauldron
 --  Mob: Goblin Alchemist
 -----------------------------------
+local ID = require("scripts/zones/Ifrits_Cauldron/IDs")
 require("scripts/globals/regimes")
+require("scripts/globals/mobs")
```

Comment:
not game breaking, but, I dont think this concrete require is needed.

Reply:
the lotto pop mechanics are in there, so its better to have it for clarity